internal/controller/drplacementcontrol.go

```diff
@@ -368,6 +368,13 @@ func (d *DRPCInstance) RunFailover() (bool, error) {
 		addOrUpdateCondition(&d.instance.Status.Conditions, rmn.ConditionAvailable, d.instance.Generation,
 			metav1.ConditionTrue, string(d.instance.Status.Phase), "Completed")
 
+		if err := d.ensureVRGManifestWork(failoverCluster); err != nil {
+			d.log.Info("Unable to ensure VRG ManifestWork on failover cluster")
+			d.setProgression(rmn.ProgressionWaitForReadiness)
```

Comment:
(nit) instead of setting the progression to `WaitForReadiness`, I would just not set it and the last progression will stay.